src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -76,6 +76,12 @@ const sortedByMeasureTypes = [
     VisualizationTypes.FUNNEL
 ];
 
+const dontSupportStackingTypes = [
```

Comment:
`unsupportedStackingTypes` would be better as we use verbs just for functions and methods